incubator/mysqlha/templates/svc.yaml

```diff
@@ -31,5 +31,7 @@ spec:
   ports:
   - name: {{ template "fullname" . }}
     port: 3306
+  - name: metrics
```

Comment:
should it be possible to add annotations to the service? This is typically used to make prometheus aware of the endpoints and scrape them

Reply:
Add metrics annotations to the service.

```
  annotations:
{{- if and (.Values.metrics.enabled) (.Values.metrics.annotations) }}
{{ toYaml .Values.metrics.annotations | indent 4 }}
{{- end }}
```